frontend/src/index.tsx

```diff
@@ -117,6 +117,11 @@ const options: HighlightOptions = {
 	version: import.meta.env.REACT_APP_COMMIT_SHA ?? '1.0.0',
 	serviceName: 'frontend',
 	otlpEndpoint: OTLP_ENDPOINT,
+	otel: {
+		instrumentations: {
+			'@opentelemetry/instrumentation-user-interaction': false,
+		},
+	},
```

Comment:
Do we want to remove this before merging?

Reply:
Not too opinionated here - Personally I don't find the traces that useful, so was going to leave it in?

Reply:
OK, cool. Thought it might be worth dogfooding since we have it set up as one of the defaults and it's in the [meta instrumentation for web](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-web).